python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
If this issue is triggered we should carefully investigate why. 2.7.9 enabled it by default, PEP 0476 [1]. I am a bit amazed that before it actually never validated certificate thus leaving an attack vector.

I guess DAS certificate was signed with CERN ROOT CA, and those certificates are installed on the system by default.

[1] https://www.python.org/dev/peps/pep-0476/